catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseDatabaseOperations.java

```diff
@@ -44,22 +52,65 @@ private static Set<String> createSysOceanBaseDatabaseNames() {
   @Override
   public String generateCreateDatabaseSql(
```

Comment:
we can make this method as parent's default implementation for mysql and oceanbase reusing

Reply:
Fixed.